apm-agent-plugins/apm-okhttp-test/src/test/java/co/elastic/apm/agent/okhttp/OkHttp3ClientVersionIT.java

```diff
@@ -0,0 +1,39 @@
+package co.elastic.apm.agent.okhttp;
+
+import co.elastic.apm.agent.TestClassWithDependencyRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class OkHttp3ClientVersionIT {
```

Comment:
[minor] maybe remove the `3` in the class name as we only test version 4.x

Reply:
It's named that way as it tests the `com.squareup.okhttp3` module. I've added some versions from the 3.x version line.